### DISCUSSION TITLE
How to enable the second skin layer for FakePlayer?

### DISCUSSION THREAD

**Discussion by DearFox** [Minestom/Minestom#989] [Help] [answered]
I experimented with FakePlayer but for some reason they don't show the second layer on the skin

**Comment by kiip1** [comment, marked as answer]
https://javadoc.minestom.net/net/minestom/server/entity/metadata/PlayerMeta.html contains the methods to toggle the second layer. (setHatEnabled, setRightSleeveEnabled, etc.) you can use FakePlayer#getMeta and cast it to PlayerMeta to access all these methods.
